Expensify/App
# Add Video Call Chat Links

### Details
<Explanation of the change or anything fishy that is going on>

This change introduces a new menu for video chats. Currently, the implementation just brings up a new video chat (zoom or google meet) and then we expect the user to share the link in the chat. 

#### Possible Improvement
It would be great to have the generated meeting link show up in the chat. This seems possible with Zoom because they have an API, but unfortunately Google Meet doesn't. I'm a bit hesitant to implement it for one but not the other, and since it was not part of the initial issue, I'm going to leave that out for now.

### Fixed Issues

Fixes https://github.com/Expensify/Expensify/issues/156264

### Tests / QA Steps


The test steps here are pretty straight-forward.
1. Go to a chat with any other person (test account or not)
2. Click the little phone icon at the top right
3. Select either zoom or google meet (they'll both want you to sign in first, so the first time you click either of them, you'll need to sign into their platform)
4. Verify that you're brought to a new video chat. 

### Tested On


### Screenshots


#### Web


<img width="406" alt="Screen Shot 2021-03-26 at 4 00 06 PM" src="https://user-images.githubusercontent.com/49007721/112700956-8da52080-8e4c-11eb-90a2-744894ead4be.png">
<img width="1482" alt="Screen Shot 2021-03-26 at 4 02 06 PM" src="https://user-images.githubusercontent.com/49007721/112700984-a0b7f080-8e4c-11eb-96dd-181df443dfae.png">

#### Mobile Web

<img width="391" alt="Screen Shot 2021-03-26 at 4 02 48 PM" src="https://user-images.githubusercontent.com/49007721/112701023-b75e4780-8e4c-11eb-84dd-6f6bcaed3806.png">

#### Desktop


<img width="1211" alt="Screen Shot 2021-03-26 at 4 03 57 PM" src="https://user-images.githubusercontent.com/49007721/112701083-e2e13200-8e4c-11eb-977e-93a4c1a0be70.png">

#### iOS


<img width="410" alt="Screen Shot 2021-03-26 at 4 08 12 PM" src="https://user-images.githubusercontent.com/49007721/112701326-90ecdc00-8e4d-11eb-8c58-714d8b60c558.png">

#### Android


<img width="562" alt="Screen Shot 2021-03-26 at 4 17 26 PM" src="https://user-images.githubusercontent.com/49007721/112701786-ccd47100-8e4e-11eb-8926-b245c88f3449.png">
